package.json

```diff
@@ -16,29 +16,34 @@
     "tiny-emitter": "^1.0.0"
   },
   "devDependencies": {
-    "babel": "^5.8.29",
-    "babelify": "^6.3.0",
+    "babel-cli": "^6.5.1",
+    "babel-core": "^6.5.2",
+    "babel-plugin-transform-es2015-modules-umd": "^6.5.0",
+    "babel-preset-es2015": "^6.5.0",
+    "babel-preset-es2015-loose": "^7.0.0",
+    "babelify": "^7.2.0",
     "bannerify": "Vekat/bannerify#feature-option",
-    "browserify": "^11.2.0",
+    "browserify": "^13.0.0",
     "chai": "^3.4.1",
+    "install": "^0.4.4",
     "karma": "^0.13.10",
-    "karma-browserify": "^4.4.0",
+    "karma-browserify": "^5.0.1",
     "karma-chai": "^0.1.0",
     "karma-mocha": "^0.2.0",
-    "karma-phantomjs-launcher": "^0.2.1",
+    "karma-phantomjs-launcher": "^1.0.0",
     "karma-sinon": "^1.0.4",
     "mocha": "^2.3.3",
-    "phantomjs": "^1.9.18",
-    "phantomjs-polyfill": "0.0.1",
+    "npm": "^3.7.3",
```

Comment:
Why would we need npm as a dependency? They will not be able to run the script without it anyways.